packages/adena-extension/src/components/warning/warning-box.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import WarningBox, { WarningBoxProps } from "./warning-box";
+
+export default {
+  title: "components/common/WarningBox",
+  component: WarningBox,
+} as ComponentMeta<typeof WarningBox>;
+
+const Template: ComponentStory<typeof WarningBox> = (args: WarningBoxProps) => (
+  <div>
+    <WarningBox {...args} />
+  </div>
+);
+
+export const ApproachPassword = Template.bind({});
+ApproachPassword.args = { type: "approachPassword", margin: "5px 5px", padding: "5px 5px" };
+
+export const ApproachPrivate = Template.bind({});
+ApproachPrivate.args = { type: "approachPrivate", margin: "5px 5px", padding: "5px 5px" };
+
+export const RevealPassword = Template.bind({});
+RevealPassword.args = { type: "revealPassword", margin: "5px 5px", padding: "5px 5px" };
+
+export const RevealPrivate = Template.bind({});
+RevealPrivate.args = { type: "revealPrivate", margin: "5px 5px", padding: "5px 5px" };
```

Comment:
It's probably a good idea to follow the new version style.
https://storybook.js.org/blog/component-story-format-3-0/

```
import WarningBox, { type WarningBoxProps } from './warning-box';
import { Meta, StoryObj } from '@storybook/react';

export default {
  title: 'components/common/WarningBox',
  component: WarningBox,
} as Meta<typeof WarningBox>;

export const ApproachPassword: StoryObj<WarningBoxProps> = {
  args: { type: 'approachPassword', margin: '5px 5px', padding: '5px 5px' },
};

export const ApproachPrivate: StoryObj<WarningBoxProps> = {
  args: { type: 'approachPrivate', margin: '5px 5px', padding: '5px 5px' },
};

export const RevealPassword: StoryObj<WarningBoxProps> = {
  args: { type: 'revealPassword', margin: '5px 5px', padding: '5px 5px' },
};

export const RevealPrivate: StoryObj<WarningBoxProps> = {
  args: { type: 'revealPrivate', margin: '5px 5px', padding: '5px 5px' },
};
```

Reply:
Thanks for the suggestion. I changed it too and it looks better.